dist/utils/text/_text.scss

```diff
@@ -50,3 +54,24 @@
 %u-align-baseline {
     vertical-align: baseline !important;
 }
+
+
+// Colour accents
+// ---
+//
+// When we need to mark text with colour
+
+.u-text-accent,
+%u-text-accent {
```

Comment:
@ry5n didn't we eventually decide not to add placeholder selectors to our utility classes? Or am I misremembering?